tests/Doctrine/Tests/ORM/Functional/QueryDqlFunctionTest.php

```diff
@@ -310,9 +312,12 @@ public function testDateAdd(string $unit, int $amount, int $expectedValue, int $
         self::assertArrayHasKey('now', $result);
         self::assertArrayHasKey('add', $result);
 
-        $diff = strtotime($result['add']) - strtotime($result['now']);
-
-        self::assertEquals($expectedValue, $diff, '', $delta);
+        self::assertEquals(
+            (new DateTimeImmutable($result['now']))->modify(sprintf('+%d %s', $amount, $unit)),
```

Comment:
Afaik CS currently prefers `\DateTimeImmutable` instead of `use` for global types, but allows both. https://github.com/doctrine/coding-standard/blob/3.0.0/lib/Doctrine/ruleset.xml#L158

Reply:
Stupid IDE config HAHAHA, will change, thanks!